package.json

```diff
@@ -477,22 +477,22 @@
         },
         {
           "command": "function.build",
-          "when": "view == knativeFunctionProjectExplorer && viewItem =~ /^(localFunctions|localDeployFunctions|localFunctionsEnablement)$/",
+          "when": "view == knativeFunctionProjectExplorer && viewItem =~ /^(localFunctions|localDeployFunctions|localFunctionsEnablement|notConnectedLocalFunctions|notConnectedLocalFunctionsEnablement)$/",
           "group": "c1@1"
         },
         {
           "command": "function.run",
-          "when": "view == knativeFunctionProjectExplorer && viewItem =~ /^(localFunctions|localDeployFunctions)$/",
+          "when": "view == knativeFunctionProjectExplorer && viewItem =~ /^(localFunctions|localDeployFunctions|notConnectedLocalFunctions)$/",
           "group": "c1@2"
         },
         {
           "command": "function.run.enablement",
-          "when": "view == knativeFunctionProjectExplorer && viewItem == localFunctionsEnablement",
+          "when": "view == knativeFunctionProjectExplorer && viewItem =~ /^(localFunctionsEnablement|notConnectedLocalFunctionsEnablement)$/",
           "group": "c1@2"
         },
         {
           "command": "function.deploy",
-          "when": "view == knativeFunctionProjectExplorer && viewItem =~ /^(localFunctions|localDeployFunctions|localFunctionsEnablement)$/",
+          "when": "view == knativeFunctionProjectExplorer && viewItem =~ /^(localFunctions|localDeployFunctions|localFunctionsEnablement|notConnectedLocalFunctionsEnablement|notConnectedLocalFunctions)$/",
```

Comment:
I would remove this. If you don't have access to a cluster you can't deploy it

Reply:
done